db/migrate_cards/20150907173833_import_production_export.rb

```diff
@@ -0,0 +1,8 @@
+# -*- encoding : utf-8 -*-
+
+class ImportProductionExport < Card::Migration
+  def up
+    import_json "production_export.json"
+    
+  end
+end
```

Comment:
why is this in a migration?  this means all this stuff will get re-imported into the production site when we run migrations (potentially overwriting changes that have happened in the meantime)